extensions/notebook/src/book/bookModel.ts

```diff
@@ -54,9 +54,13 @@ export class BookModel implements azdata.nb.NavigationProvider {
 
 			let p = path.join(workspacePath, '**', '_data', 'toc.yml').replace(/\\/g, '/');
 			let tableOfContentPaths = await glob(p, { deep: maxDepth });
-			this._tableOfContentPaths = this._tableOfContentPaths.concat(tableOfContentPaths);
-			let bookOpened: boolean = this._tableOfContentPaths.length > 0;
-			vscode.commands.executeCommand('setContext', 'bookOpened', bookOpened);
+			if (tableOfContentPaths.length > 0) {
+				this._tableOfContentPaths = this._tableOfContentPaths.concat(tableOfContentPaths);
+				vscode.commands.executeCommand('setContext', 'bookOpened', true);
+			} else {
+				vscode.window.showErrorMessage(loc.errBookInitialize);
```

Comment:
i'm confused by this line. does this mean that there will be an error when anyone opens up ADS and a book isn't found?

Reply:
No, only when they open a book and then if the toc file isn't found.